report/delayedusers/classes/report.php

```diff
@@ -131,7 +131,7 @@ public function get_submissions_sql() {
         $submissiontable = 'SELECT userid, surveyproid
```

Comment:
Sorry but... what do you want to calculate with this query? Grouping by userid... what? If you group, you need an aggregation function (max, mix, count...). So it's not clear what is being attempted here.